hackathon_site/dashboard/frontend/src/components/general/ProductOverview/ProductOverview.js

```diff
@@ -0,0 +1,196 @@
+import React from "react";
+import Typography from "@material-ui/core/Typography";
+import Link from "@material-ui/core/Link";
+import Button from "@material-ui/core/Button";
+import MenuItem from "@material-ui/core/MenuItem";
+import FormControl from "@material-ui/core/FormControl";
+import Select from "@material-ui/core/Select";
+import LaunchIcon from "@material-ui/icons/Launch";
+import InputLabel from "@material-ui/core/InputLabel";
+import Chip from "@material-ui/core/Chip";
+import styles from "./ProductOverview.module.scss";
+import Alert from "@material-ui/lab/Alert";
+
+import { Formik } from "formik";
+
+const createQuantityList = (number) => {
+    let entry = [];
+
+    for (let i = 1; i <= number; i++) {
+        let valueInString = i + "";
+        entry.push(
+            <MenuItem key={i} role="quantity" value={valueInString}>
+                {i}
+            </MenuItem>
+        );
+    }
+
+    return entry;
+};
+
+export const CartForm = ({
+    availableQuantity,
+    handleSubmit,
+    handleChange,
+    requestFailure,
+    values: { quantity },
+}) => {
+    if (!handleSubmit) {
+        return false;
+    }
+
+    return (
+        handleSubmit && (
```

Comment:
This is redundant with the check above, but I would remove it as well. There shouldn't be any real case where this is created without `handleSubmit`.

Reply:
In the figma there are product overviews without an add cart button so I added this for those cases.

Reply:
Ah, good catch! In those cases, we'd actually want to selectively render the entire form in the `ProductOverview` section, rather than at this level. `ProductOverview` would just have 

```javascript
{addToCart && <CartForm ... />}
```

The reasoning being we can save having to process the formik logic if we know that the form itself isn't going to be there.